tests/rt.sh

```diff
@@ -786,10 +786,10 @@ case ${MACHINE_ID} in
     COMPILE_QUEUE="batch"
 
     PARTITION=
-    dprefix="/scratch1/NCEPDEV"
-    DISKNM="/scratch2/NAGAPE/epic/UFS-WM_RT"
-    STMP="${dprefix}/stmp4"
-    PTMP="${dprefix}/stmp2"
+    dprefix="/scratch4/NCEPDEV"
```

Comment:
would you be willing to use a similar form here, and for all machine cases, the same way it is done for gaeac5:
dprefix=${dprefix:-/gpfs/f5/${ACCNR}/scratch/${USER}}